### DISCUSSION TITLE
Add `full` variant for radius config

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#1728] [Feature requests]
`full` variant is needed for make `pill` buttons (and circles, of course). Reference [here](https://tailwindcss.com/docs/border-radius#pill-buttons)
```
// Default border-radius used for most elements
  defaultRadius: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'full' | number;
```
`full` can have value of `9999px`

**Comment by rtivital** [comment]
You can just use `defaultRadius: 9999`, there is no need for `full` alias for this value
